.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,38 @@
+name: Npm.js deploy
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'npm'
+          registry-url: 'https://registry.npmjs.org'
+      - run: echo "VERSION=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+      - run: echo ${VERSION}
+
+      - run: npm ci
+
+      - name: Build the package
+        run: npm run build
+
+      - name: Define npm tag
+        run: |
+            if [[ ${VERSION} == *"alpha"* || ${VERSION} == *"beta"* || ${VERSION} == *"rc"* ]]; then echo "NPM_TAG=next"; else echo "NPM_TAG=latest"; fi >> $GITHUB_ENV
+            echo ${NPM_TAG}
+
+      - name: Version the package
+        run: npm version ${VERSION} --no-git-tag-version --save --verbose
+        working-directory: dist
+
+      - name: Publish the package
+        run: npm publish --tag ${NPM_TAG}
+        working-directory: dist
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
```

Comment:
Looking at the two packages going to dist, this is unlikely to work. If this wasn't initially copied over from igniteui-angular, I suggest copying both steps:
https://github.com/IgniteUI/igniteui-angular/blob/81e84062f2a133f7d1c7f8b903a25863df2a3b77/.github/workflows/npm-publish.yml#L58-L68